tiledb/tests/test_pandas_dataframe.py

```diff
@@ -28,6 +28,9 @@
 
 if not has_pandas():
     pytest.skip("pandas>=1.0,<3.0 not installed", allow_module_level=True)
+else:
+    pd = pytest.importorskip("pandas")
```

Comment:
This doesn't make sense to me - what are we trying to do here? The original line worked correctly.

Reply:
original code is not checking for the python version. New code is not importing pandas, even it exists.

Reply:
```suggestion
    import pandas as pd
```

The above call to `has_pandas()`+`pytest.skip()` already skips the test file if pandas isn't installed (or if its the wrong version). Thus if `has_pandas()` evaluates to `True`, you can simply import pandas directly